src/applications/gi/components/search/TypeOfInstitutionFilter.jsx

```diff
@@ -35,9 +34,21 @@ class TypeOfInstitutionFilter extends React.Component {
             target="_blank"
             rel="noopener noreferrer"
           >
-            Learn More
+            Learn More)
+            <br />
           </a>
-          )
+          {this.props.category === 'vettec' && (
+            <span className="vads-u-margin-x--neg5">
+              {' '}
```

Comment:
just want to point this out and make sure it's not an oversight

Reply:
It is not an oversight, needed it to style and render logo.